### DISCUSSION TITLE
how to convert a double field to string field remove decimal and zero

### DISCUSSION THREAD

**Discussion by jefunes** [duckdb/duckdb#12508] [General]
in a r script why when converting a double field to a character field, it adds a dot and a decimal to a duckdb table, how can i fix this?

**Comment by soerenwolfers** [comment]
Check `format()` and `printf()` at https://duckdb.org/docs/sql/functions/char.html#fmt-syntax

Alternatively, cast to [integer](https://duckdb.org/docs/sql/data_types/numeric.html#integer-types) or [fixed point decimal](https://duckdb.org/docs/sql/data_types/numeric.html#fixed-point-decimals) type.
